x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface CspmUsage {
+  indices: CspmIndicesStats;
+}
+
+export interface CspmIndicesStats {
+  findings: IndexStats | {};
+  latest_findings: IndexStats | {};
+  score: IndexStats | {};
+}
```

Comment:
I accept your suggestion and then got a functional test error for the test file: 
`x-pack/test/api_integration/apis/telemetry/telemetry.ts`

```
Error: The telemetry schemas in 'x-pack/plugins/telemetry_collection_xpack/schema/' are out-of-date, please update it as required: [stack_stats.kibana.plugins.cloud_security_posture.indices.findings]: expected a plain object value, but found [null] instead.
```

Reply:
@CohenIdo it seems you had that error because you had changed your schema without a proper migration process.
Try to start from a clean development environment and check if you still get that issue.

If that works, maybe instead of `null`, `undefined` is a better option?

Reply:
@kfirpeled there is a script, `node scripts/telemetry_check --fix`, that takes care for update the scheme, it failed with null, but definitely it's worth to test it with `undefined`